packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -158,10 +164,18 @@ export class BlobManager {
         private readonly runtime: IBlobManagerRuntime,
         stashedBlobs: IPendingBlobs = {},
     ) {
-        this.logger = ChildLogger.create(this.runtime.logger, "BlobManager");
+        this.mc = loggerToMonitoringContext(ChildLogger.create(this.runtime.logger, "BlobManager"));
```

Comment:
This code should not be part of this PR as this is a specific feature level work. Tyler is doing this in a separate PR.

Reply:
I think Tyler has checked it in. Not sure why it shows as my changes.